d3a_api_client/setups/test_create_aggregator.py

```diff
@@ -28,13 +29,13 @@ def on_market_cycle(self, market_info):
             return
 
         for device_event in market_info["content"]:
-            if "available_energy_kWh" in device_event["device_info"] and \
+            if key_in_dict_and_not_none(device_event, "available_energy_kWh") and \
```

Comment:
It would be nice if you can separate the checks into private functions like [This one](https://github.com/gridsingularity/d3a-api-client/blob/80a074c5e3543639a69afa925f7c9a3d7797d154/integration_tests/test_aggregator_batch_commands.py#L198)

Reply:
No need, there is already a `key_in_dict_and_not_none_and_greater_than_zero` in `d3a-interface`, will re-use it